packages/die.vm/die.vm.nuspec

```diff
@@ -2,11 +2,11 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>die.vm</id>
-    <version>3.07.20230523</version>
+    <version>3.07.20230925</version>
     <authors>Hellsp@wn, horsicq</authors>
     <description>Detect It Easy, or abbreviated "DIE" is a program for determining types of files.</description>
     <dependencies>
-      <dependency id="common.vm" />
+      <dependency id="common.vm" version="0.0.0.20230925" />
```

Comment:
if we pin the dependency do we have to update manually?

Reply:
nuspec version ranges are a bit confusing:
![Screenshot 2023-09-27 at 13 29 50](https://github.com/mandiant/VM-Packages/assets/16052290/99743c90-031e-4e8f-9001-c2620cf761b4)
So this is requesting version >= 0.0.0.20230925 to ensure that the argument format is compatible. So that if an older `common.vm` is installed it is upgraded when installing this version of `die.vm`.

We don't need to update it manually when we update `common.vm`.

Reply:
ok, perfect, that should work nicely then!